sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/setup/SetupAccountActivity.kt

```diff
@@ -203,9 +213,10 @@ class SetupAccountActivity : DuckDuckGoActivity(), SetupFlowListener {
             SETUP_COMPLETE,
         }
 
-        internal fun intent(context: Context, screen: Screen): Intent {
+        internal fun intent(context: Context, screen: Screen, source: String?): Intent {
```

Comment:
NIT: As this is not really part of the changes, this can also be trasnformed to use `globalActivityStarter.startIntent`